### ISSUE TITLE
[TF24 NSC] Literature review of Non-structural carbohydrate storage

### ISSUE THREAD

**Issue by dfalster** [traitecoevo/plant#419]
Data

- Adams HD et al. (2017) A multi-species synthesis of physiological mechanisms in drought-induced tree mortality. Nature Ecology & Evolution 1: 1285–1291. DOI: [10.1038/s41559-017-0248-x](https://doi.org/10.1038/s41559-017-0248-x)
- Dai Y et al. (2018) Relative contributions of hydraulic dysfunction and carbohydrate depletion during tree mortality caused by drought. AoB PLANTS 10: plx069. DOI: [10.1093/aobpla/plx069](https://doi.org/10.1093/aobpla/plx069)
- El Omari B (2022) Accumulation versus storage of total non-structural carbohydrates in woody plants. Trees 36: 869–881. DOI: [10.1007/s00468-021-02240-6](https://doi.org/10.1007/s00468-021-02240-6)
- Granda E& Camarero JJ (2017) Drought reduces growth and stimulates sugar accumulation: new evidence of environmentally driven non-structural carbohydrate use. Tree Physiology 37: 997–1000. DOI: [10.1093/treephys/tpx097](https://doi.org/10.1093/treephys/tpx097)
- Klein T et al. (2014) Drought stress, growth and nonstructural carbohydrate dynamics of pine trees in a semi-arid forest. Tree Physiology 34: 981–992. DOI: [10.1093/treephys/tpu071](https://doi.org/10.1093/treephys/tpu071)
- Lloret F et al. (2018) Non-structural carbohydrate dynamics associated with drought-induced die-off in woody species of a shrubland community. Annals of Botany 121: 1383–1396. DOI: [10.1093/aob/mcy039](https://doi.org/10.1093/aob/mcy039)
- Martínez-Vilalta J et al. (2016) Dynamics of non-structural carbohydrates in terrestrial plants: a global synthesis. Ecological Monographs 86: 495–516. DOI: [10.1002/ecm.1231](https://doi.org/10.1002/ecm.1231)
- Rosas T et al. (2013) Dynamics of non-structural carbohydrates in three Mediterranean woody species following long-term experimental drought. Frontiers in Plant Science 4. DOI: [10.3389/fpls.2013.00400](https://doi.org/10.3389/fpls.2013.00400)
- Santos M et al. (2021) Whole plant water status and non‐structural carbohydrates under progressive drought in a Caatinga deciduous woody species. Trees 35: 1257–1266. DOI: [10.1007/s00468-021-02113-y](https://doi.org/10.1007/s00468-021-02113-y)
- Signori-Müller C et al. (2021) Non-structural carbohydrates mediate seasonal water stress across Amazon forests. Nature Communications 12: 2310. DOI: [10.1038/s41467-021-22378-8](https://doi.org/10.1038/s41467-021-22378-8)
- Thompson RA et al. (2023) No carbon storage in growth-limited trees in a semi-arid woodland. Nature Communications 14: 1959. DOI: [[10.1038/s41467-023-37577-8](https://doi.org/10.1038/s41467-023-37577-8)
- Urrutia-Jalabert R et al. (2020) Low Growth Sensitivity and Fast Replenishment of Non-structural Carbohydrates in a Long-Lived Endangered Conifer After Drought. Frontiers in Plant Science 11. DOI: [10.3389/fpls.2020.00905](https://doi.org/10.3389/fpls.2020.00905)

Models

- Jones S et al. (2020) The impact of a simple representation of non-structural carbohydrates on the simulated response of tropical forests to drought. Biogeosciences 17: 3589–3612. DOI: [10.5194/bg-17-3589-2020](http://doi.org/10.5194/bg-17-3589-2020)
- Cho N et al. (2022) Modeling temporal variations of non-structural carbohydrate (NSC) storages across biomes. Forest Ecology and Management 508: 120033. DOI: [10.1016/j.foreco.2022.120033](http://doi.org/10.1016/j.foreco.2022.120033)
- Cho N et al. (2022) Recent field findings and modeling on non-structural carbohydrates (NSCs): How to synthesize? Ecological Informatics 70: 101695. DOI: [10.1016/j.ecoinf.2022.101695](http://doi.org/10.1016/j.ecoinf.2022.101695)
- D’Andrea E et al. (2021) Unravelling resilience mechanisms in forests: role of non-structural carbohydrates in responding to extreme weather events. Tree Physiology 41: 1808–1818. DOI: [10.1093/treephys/tpab044](http://doi.org/10.1093/treephys/tpab044)
- Fang J et al. (2020) A physiological model for predicting dynamics of tree stem-wood non-structural carbohydrates. Journal of Ecology 108: 702–718. DOI: [10.1111/1365-2745.13274](http://doi.org/10.1111/1365-2745.13274)
- Hartmann H et al. (2018) Identifying differences in carbohydrate dynamics of seedlings and mature trees to improve carbon allocation in models for trees and forests. Environmental and Experimental Botany 152: 7–18. DOI: [10.1016/j.envexpbot.2018.03.011](http://doi.org/10.1016/j.envexpbot.2018.03.011)
- Stefaniak EZ et al. (2024) Optimal carbon storage during drought. Tree Physiology tpae032. DOI: [10.1093/treephys/tpae032](http://doi.org/10.1093/treephys/tpae032)

**Comment by aornugent** [comment]
Are you taking submissions?

https://www.nature.com/articles/s41467-023-37577-8

**Comment by dfalster** [comment]
Yes absolutely. This is very insightful and helpful, as it suggests Non-structural carbohydrate storage aren't the solution for surviving drought in arid environments. According to this ref, variable leaf area will be more important. As it allows the plant to gradually wind back it's leaf area.

**Comment by dfalster** [comment]
Goals

1. Assess empirical evidence of NSCs, how much and when ? Do they vary with
- species, tissues
- environmental conditions: drought, light
- when are they deployed
- what is the evidence they defer mortality?

2. How have people modelled this? Is there an approach we can repurpose?
- what are our core requirements
- what is the simplest representation we can use?

**Comment by aornugent** [comment]
Uni, D., Sheffer, E., Winters, G. et al. Peak photosynthesis at summer midday in Acacia trees growing in a hyper-arid habitat. Trees 37, 255–267 (2023). https://doi.org/10.1007/s00468-022-02344-7
